LoveAndHiphop/LoveAndHiphop/DetailViewController.swift

```diff
@@ -252,42 +260,6 @@ class DetailViewController: UITableViewController {
 
     func updateProfile() {
 
-        self.photoScrollView.delegate = self
```

Comment:
This is why the bug happened. Since I was reloading this code as well after the like/unlike button was clicked, another image was written above the existing image. After removing this logic, the bug has been fixed.